### PR TITLE
Initial role PR

### PR DESCRIPTION
**Pull request by haslersn**
* [x] README.md

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+galaxy_info:
+  author: haslersn
+  description: Installs Jitsi Meet; see https://github.com/jitsi/jitsi-meet
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: AGPLv3
```
Comment (Mr-Pi) [inline meta/main.yml:7]: In the past we decided to use the _Creative Commons Attribution Share Alike 4.0 International
_ license.

File: tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  apt_key:
+   url: https://download.jitsi.org/jitsi-key.gpg.key
+   id: EF8B479E2DC1389C
+
+- name: Add apt repository
+  apt_repository:
+    repo: deb https://download.jitsi.org stable/
+    update_cache: True
+
+- name: Preseed debconf
```
Comment (Mr-Pi) [inline tasks/main.yml:12]: Does ansible display a shell warning here?

File: tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  shell: echo "jitsi-videobridge jitsi-videobridge/jvb-hostname string {{ jitsi_meet_hostname }}" | debconf-set-selections
+  changed_when: False
+
+- name: Install jitsi-meet
+  apt:
+    name:
+      - apt-transport-https
+      - apt-utils
```
Comment (Mr-Pi) [inline tasks/main.yml]: Are those two packages required?

**Review by Mr-Pi — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 2 above)
Comment (haslersn) [reply tasks/main.yml:12]: No

File: tasks/main.yml (hunk 3 above)
Comment (haslersn) [reply tasks/main.yml]: Apparently not, so I removed them in my force push just now.

**Review by SlothOfAnarchy — APPROVED** [review]

**Review by Mr-Pi — APPROVED** [review]

**Pull request merged**